src/absil/illib.fs

```diff
@@ -969,7 +969,7 @@ type Map<'Key,'Value when 'Key : comparison> with
 
     member x.Values = [ for (KeyValue(_,v)) in x -> v ]
     member x.Elements = [ for kvp in x -> kvp ]
-    member x.AddAndMarkAsCollapsible (kvs: _[])   = (x,kvs) ||> Array.fold (fun x (KeyValue(k,v)) -> x.Add(k,v))
+    member x.AddAndMarkAsCollapsible (kvs: _[])   = x.AddRange(kvs)
```

Comment:
Is there a compiler switch which allows me to use the old code for building the proto compiler?